keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/jackson/DefaultSecurityGroupRuleDeserializer.kt

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.keel.ec2.jackson
+
+import com.netflix.spinnaker.keel.api.ec2.CidrRule
+import com.netflix.spinnaker.keel.api.ec2.CrossAccountReferenceRule
+import com.netflix.spinnaker.keel.api.ec2.ReferenceRule
+import com.netflix.spinnaker.keel.api.ec2.SecurityGroupRule
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.boot.jackson.JsonComponent
+
+@JsonComponent
+//@ConditionalOnMissingBean(SecurityGroupRuleDeserializer::class)
```

Comment:
Unfortunately, this did not work. Spring never instantiates this bean, even when there's not another one that's a sub-class of `SecurityGroupRuleDeserializer`. It's something to do with timing/ordering, as far as I understand from the docs and StackOverflow questions.